tests/features/steps/then.py

```diff
@@ -86,6 +81,14 @@ def step_impl(context):
         if dat != re['hits'][0]['data']:
             raise UnexpectedBehaviourError('Secure Timeseries data not found in response')
 
+    if context.type == 'authorised_id_multiple':
+        dat = {"hello": "india"}
+        dat1 = {"hello": "world"}
+        re = context.response.json()
+        print(re)
+        if dat1 != re['hits'][0]['data']:
```

Comment:
Better to
* check the length of the response
* and then iterate through them trying to find the desired results

Hardcoding index values is a bad idea

Reply:
Made changes